DOC-2581: Uploadcare new plugin documentation. (#3515)
tinymce/tinymce-docs: modules/ROOT/partials/toolbar-button-ids/uploadcare-toolbar-buttons.adoc

[cols="1,3",options="header"]
|===
|Identifier |Description

> | `+uploadcare+` | Inserts a `<p>` tag with the `+contenteditable="false"+` attribute, containing a placeholder image. This placeholder supports image uploads via drag-and-drop, file selection, or by providing a URL.

Reviewer: I don't thing we need to go deep on the details here.

```suggestion
| `+uploadcare+` | Inserts a image placeholder element. This placeholder supports image uploads via drag-and-drop, file selection, or by providing a URL.
```

Reviewer 2: do they need to apply the same change to: modules/ROOT/partials/menu-item-ids/uploadcare-menu-items.adoc ?

Author: Updated in last commit for both `menu-items` and `toolbar-buttons`
